tools/wdio/runner.mjs

```diff
@@ -28,11 +29,25 @@ const configFilePath = path.join(
   `wdio.conf_${crypto.randomBytes(16).toString('hex')}.mjs`
 )
 
-if (['trace', 'debug', 'info'].indexOf(wdioConfig.logLevel) > -1) {
+if (args.verbose) {
   console.log(`Writing wdio config file to ${configFilePath}`)
 }
 
-fs.ensureDirSync(path.dirname(configFilePath))
+// Clean output directories
+fs.emptyDirSync(path.dirname(configFilePath))
+
+const lambdaTestOutputDir = path.join(
+  path.resolve(__dirname, '../../'),
+  '.lambdatest'
+)
+fs.ensureDirSync(lambdaTestOutputDir)
+const lambdaTestTunnelLog = path.join(
+  lambdaTestOutputDir,
+  'lt.log'
+)
+if (fs.pathExistsSync(lambdaTestTunnelLog)) {
+  fs.removeSync(lambdaTestTunnelLog)
+}
```

Comment:
Shouldn't do this. Leave it manual. It may also cause issues when executing 2 runners simul

Reply:
You mean executing wdio with the `-T` flag in two different terminals?

Reply:
Honestly, I added this to make capturing tunnel logs easier so I could give the right log to LT team. IDC either way but I would just point out that the LT plugin also downloads the tunnel binary to the same directory so I wouldn't advise running wdio in two different tunnels with the `-T` flag.